src/components/player/Player.vue.js

```diff
@@ -0,0 +1,91 @@
+export default {
+  name: 'player',
+  props: {
+    data: {}
+  },
+  data() {
+    return {
+      id: {
+        'Progress': '.progress-bar',
+        'Duration': '.max-time',
+        'Pause':    '.button-pause',
+        'Play':     '.button-play',
+        'Audio':    'audio'
+      },
+      el: {},
+    }
+  },
+  methods: {
+    _pop(object) {
+      let key = Object.keys(object).pop()
+      return {
+        key: key,
+        value: object[key],
+        remainder: (delete object[key], object)
```

Comment:
Isn't it useless to return the remainder, as `delete` mutates the object passed to it?